README.md

```diff
@@ -1,5 +1,59 @@
-# curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# Project Name
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+> My Personal Portfolio.
+
+
+## Built With
+
+- HTML5
+- CSS3
+
+## Live Demo
+
+[mgmediaweb.github.io/portfolio](https://mgmediaweb.github.io/portfolio)
```

Comment:
- Kindly avoid using links to other projects. The live demo link should present the live version of the content in this repository. Please pay attention to make all necessary changes before requesting a new review.